src/test/java/de/tarent/maven/plugins/pkg/MvnPkgPluginDefaultDependencyLineTest.java

```diff
@@ -0,0 +1,40 @@
+package de.tarent.maven.plugins.pkg;
```

Comment:
File Comment:

Can we somehow add a test that shows that if one doesn't specify a default dependency line then `default-jre` will be added on debian wheezy?